README.md

```diff
@@ -15,11 +15,17 @@
 
 This repository contains the scripts for building OpenSearch and OpenSearch Dashboards distributions.
 
-* [OpenSearch Bundle Workflow](bundle-workflow/README.md)
+- [OpenSearch Bundle Workflow](bundle-workflow/README.md)
+
+### Organization
+
+OpenSearch's centralized DevOps team manages centralized build and release processes.
```

Comment:
Use "centralized" once, e.g. "OpenSearch DevOps team manages a centralized build and release process."